packages/theme/css/foundations/size.css

```diff
@@ -19,7 +18,6 @@
   --salt-size-indicator: 2px;
   --salt-size-selectable: 14px;
   --salt-size-bar-strong: 8px;
-  /* --salt-size-bar-small: 4px; */
```

Comment:
this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable